src/app/beer_garden/db/mongo/pruner.py

```diff
@@ -5,16 +5,18 @@
 
 from brewtils.stoppable_thread import StoppableThread
 from mongoengine import Q
+from mongoengine.queryset.visitor import Q as Q2
 
 from beer_garden.db.mongo.models import File, RawFile, Request
 
 
 class MongoPruner(StoppableThread):
-    def __init__(self, tasks=None, run_every=None):
+    def __init__(self, tasks=None, run_every=None, ttl=None):
```

Comment:
`ttl` is a bit too generic in this context.  Maybe call it `cancel_threshold` or something?

Also, the name becomes marginally less critical if there was a proper docstring for the class explaining what the attributes are.  Let's take the opportunity here and write up a docstring for this class so that you can document the new one.  It looks like we have literally never written a docstring for a class in the code base (shame on us), but for the other places where docstrings exist we strive to follow the google style guide.  See the section on class docstrings [here](https://google.github.io/styleguide/pyguide.html#384-classes).